contrib/grpcplugins/action/checkout/main.go

```diff
@@ -61,6 +62,13 @@ func (p *checkoutPlugin) Stream(q *actionplugin.ActionQuery, stream actionplugin
 		authOption = repo.WithSSHAuth([]byte(key.Private))
 	}
 
+	// Create directory
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		if err := os.MkdirAll(path, os.ModePerm); err != nil {
```

Comment:
ModePerm = [fs](https://pkg.go.dev/io/fs).[ModePerm](https://pkg.go.dev/io/fs#ModePerm) // Unix permission bits, 0o777

not sure it is the right FileMode